inception-project/inception
# #802 - Rendering annotations pagewise

**What's in the PR**
* add pagewise annotation rendering

**How to test manually**
* open a PDF document in PDF editor
* scroll to pages and see rendering

**Automatic testing**

**Documentation**
